python/python/bystro/search/utils/beanstalkd.py

```diff
@@ -0,0 +1,233 @@
+"""TODO: Add description here"""
```

Comment:
it seems that beanstalkd.py provides functionality intended to be shared by all python workers.  If so, it should go in its own top-level directory in the python section of this repo rather than live in `search` code.  Otherwise we'll have `ancestry` (and every other worker) rummaging around in `search` code for its imports.